### PR TITLE
defaultconfig: bump http2-max-streams-per-connection to 2000

### PR DESCRIPTION
**Pull request by mfojtik**
To mitigate https://github.com/kubernetes/kubernetes/issues/74412

/cc @sttts 
/cc @deads2k 
/cc @tnozicka

### CODE REVIEW

**Comment by mfojtik** [comment]
@sttts @deads2k looks like this flag makes apiserver to crash loop...

**Comment by deads2k** [comment]
> @sttts @deads2k looks like this flag makes apiserver to crash loop...

got a log?

File: bindata/v3.11.0/kube-apiserver/defaultconfig.yaml [hunk 1]
```diff
@@ -32,6 +32,8 @@ apiServerArguments:
   - PersistentLocalVolumes=false # disable local volumes for 4.0, owned by sig-storage/hekumar@redhat.com
   minimal-shutdown-duration:
   - 3s # give SDN some time to converge
+  http2-max-streams-per-connection:
+  - 2000  # recommended is 1000, but we need to mitigate https://github.com/kubernetes/kubernetes/issues/74412
```
Comment (deads2k) [inline bindata/v3.11.0/kube-apiserver/defaultconfig.yaml]: "2000" to make sure it serializes as a string

Comment (mfojtik) [reply bindata/v3.11.0/kube-apiserver/defaultconfig.yaml]: i bet that was it! good catch

**Pull request merged**